CommonTools/TrackerMap/src/TrackerMap.cc

```diff
@@ -23,6 +23,9 @@
 #include "TArrow.h"
 #include "TLegend.h"
 #include "TH1F.h"
+#include "TH2Poly.h"
+#include "TGraph.h"
+#include "TPaletteAxis.h"
```

Comment:
Just to note that this dependence would have required the addition of `roothistpainter` dependence in the `BuildFile.xml` to keep UBSAN build functional. I have a fix in https://github.com/cms-sw/cmssw/pull/42095.